frontend/packages/console-shared/src/components/popover/SizeAwarePopover.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import Measure from 'react-measure';
+import { Popover, Button } from '@patternfly/react-core';
+import './SizeAwarePopover.scss';
+
+type SizeAwarePopoverProps = {
+  content: string;
```

Comment:
Add prop `lineClamp?: number`.
Even though for our use case we want a value of 5. I think 1 line is a better default value